modules/moa-faq-installing.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/moa-faq.adoc
+
+
+[id="moa-faq-installing"]
+
+= Installation troubleshooting
+
+== Inspect install or uninstall logs
+
+Run the following command to generate install logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs install <my_cluster_name>
+----
+
+Run the following command to generate uninstall logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs uninstall <my_cluster_name>
+----
+
+== Verify your AWS account does not have an SCP
+
+Run the following command to verify your AWS account has the correct permissions.
+
+[source,terminal]
+----
+$ moactl verify permissions
+----
+
+If you receive any errors, double check to ensure than an link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_type-auth.html#orgs_manage_policies_scp[SCP] is not applied to your AWS account. If you are required to use an SCP, see link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
+
+== Verify your AWS account and quota
+
+Run the following command to verify you have the available quota on your AWS account. 
+
+[source,terminal]
+----
+$ moactl verify quota
```

Comment:
Here perhaps link to the output of https://issues.redhat.com/browse/OSDOCS-1444 and https://issues.redhat.com/browse/SDA-2838

Reply:
I'm gonna track this on the quota doc card as I've got to take a second pass at it.
https://issues.redhat.com/browse/OSDOCS-1444?focusedCommentId=15203223&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15203223